weblate/api/tests.py

```diff
@@ -46,6 +46,8 @@
 
 TEST_PO = get_test_file("cs.po")
 TEST_POT = get_test_file("hello-charset.pot")
+TEST_DOC = get_test_file("short_description.txt")
```

Comment:
Please use something more realistic for testing, see comment below.

```suggestion
TEST_DOC = get_test_file("cs.html")
```